visualization/textlog.py

```diff
@@ -0,0 +1,37 @@
+import pygame
+
+text_log = []
+
+text_log_surface = None
+max_lines_to_show = 0
+
+pygame.font.init()
+log_font = pygame.font.Font("visualization/opti.pcf", 16)
+
+line_height = 11
+padding_from_edges = 4
+
+color_fg = (150, 150, 150)
+
+def init_text_log_surface(log_size):
```

Comment:
still not super convinced about the whole module-as-singleton thing... but i can do the refactor if you don't wanna :p

Reply:
nah i'll refactor it, just need to think about it a bit more

Reply:
my thought is you just have a TextLog class with these functions as its methods - you could even start with having a singleton instance of it (rather than Surface) in this module, if that helps get the ball rolling